quattor/blockdevices.pan

```diff
@@ -155,12 +155,20 @@ type blockdevices_vxvm_type = {
     "size" ? long
 };
 
+# TODO, remove after a while, was used instead of more generic dummy_type
 @documentation{
     TMPFS devices (dummy devices)
 }
 type blockdevices_tmpfs_type = {
 };
 
+@documentation{
+    Dummy devices (e.g. tmpfs, proc)
+}
+type blockdevices_dummy_type = {
```

Comment:
Maybe, s/dummy/empty/ here? Or "special" ? Or placeholder ?

My reasoning is that these still have an fs_spec, it's just the way that Quattor works that requires them to have an entry in structure_blockdevices that forces us to have this type? So to call them of type "dummy" is slightly misleading because we don't actually create anything, this is all quattor internal plumbing.

Reply:
In the context of https://github.com/quattor/ncm-lib-blockdevices/pull/94#pullrequestreview-655827634 then maybe this is the right term.